java-client/src/main/java/org/opensearch/client/opensearch/indices/IndexSettings.java

```diff
@@ -1066,7 +1066,7 @@ protected void serializeInternal(JsonGenerator generator, JsonpMapper mapper) {
 
 		}
 		if (this.knnAlgoParamEfSearch != null) {
```

Comment:
Not related to this change, but it seems like a bunch of KNN index settings are missed:

```
"settings" : {
        "index": {
            "knn": true,
            "knn.algo_param.m":90,
            "knn.algo_param.ef_construction":200,
            "knn.algo_param.ef_search":300
        }
    },
```

https://github.com/opensearch-project/k-NN/blob/main/RFC.md?plain=1